src/sentry/static/sentry/app/components/events/contexts/operatingSystem/operatingSystem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import ContextBlock from 'app/components/events/contexts/contextBlockV2';
+
+import getOperatingSystemKnownData, {
+  OperatingSystemKnownData,
+} from './getOperatingSystemKnownData';
+
+type Props = {
+  data?: OperatingSystemKnownData;
+};
+
+const OperatingSystem = ({data}: Props) => {
+  if (data === undefined || data === null) {
```

Comment:
You might want to consider this utility we got:
https://github.com/getsentry/sentry/blob/680efc930955f25a932a0d6aad0f39db372db6b0/src/sentry/static/sentry/app/utils.tsx#L115-L117